monitors/CRMonitor.js

```diff
@@ -10,12 +10,16 @@ class CRMonitor {
   //    address: "0x12345",
   //    crAlert: 150 },
   // ...];
-  constructor(logger, expiringMultiPartyClient, walletsToMonitor) {
+  // @param priceFeed offchain price feed used to track the token price.
```

Comment:
Can you document the other params, similar to the liquidator constructor comments?

Reply:
Done.